src/he/ssl_ctx.h

```diff
@@ -342,6 +342,8 @@ bool he_ssl_ctx_is_inside_write_cb_set(he_ssl_ctx_t *ctx);
  * Linux this would usually be a UDP socket.
  */
 void he_ssl_ctx_set_outside_write_cb(he_ssl_ctx_t *ctx, he_outside_write_cb_t outside_write_cb);
+void he_ssl_ctx_set_outside_write_ex_cb(he_ssl_ctx_t *ctx,
```

Comment:
Please add comments for this and we can update `he_ssl_ctx_set_outside_write_cb` comment to indicate that it is deprecated

Reply:
Done.